__tests__/integration.test.js

```diff
@@ -0,0 +1,37 @@
+const express = require("express");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("../db/data/test-data/index.js");
+const app = require("../db/app");
+
+beforeEach(() => {
+  return seed({ articleData, commentData, topicData, userData });
+});
+
+afterAll(() => db.end());
+
+describe("GET /api/topics", () => {
+  test("responds with a 200 status code", () => {
+    return request(app).get("/api/topics").expect(200);
+  });
+  test("returns an array of topic objects with 'slug' and 'description' properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
```

Comment:
Your responses should be returned in an object with a key for some context about the data 

I think case, i'd probably expect `topics`